extension/README.md

```diff
@@ -164,6 +153,30 @@ as the `registries` array in `extensions.private.json`.
 You can use the **Private Extensions: Add Registry...** and
 **Private Extensions: Remove Registry** commands to quickly edit this setting.
 
+#### Custom Channels
+It is possible to create tracking channels by using npm dist-tags when
+publishing a private extension. The tracked channel is used to determine when
+extension updates are available.
+
+*Tracking a Channel*<br>
+To specify a channel to track for an extension, add it to the `privateExtensions.channels` settings object as shown in the example below:
+```
+"privateExtensions.channels" :{
+    "extensionOne": "insiders",     // Tracks the 'insiders' dist-tag
```

Comment:
Note that if we fix channels to use the full extension ID, this needs to be updated.